charts/traction/templates/ui/configmap.yaml

```diff
@@ -14,6 +14,7 @@ data:
   FRONTEND_INNKEEPER_SHOW_ADMIN: {{ .Values.ui.oidc.showInnkeeperAdminLogin | quote }}
   FRONTEND_TENANT_SHOW_WRITABLE_COMPONENTS: {{ .Values.ui.oidc.showWritableComponents | quote }}
   FRONTEND_SHOW_OIDC_RESERVATION_LOGIN: {{ .Values.ui.showOIDCReservationLogin | quote }}
+  FRONTEND_TENANT_CUSTOM_RESERVATION_FORM: {{ .Values.ui.oidc.customReservationForm | quote }}
```

Comment:
This isn't used anywhere?

Reply:
Same as above.

Reply:
But where is this env var used? I just pulled the code and I don't see it anywhere? Aren't you using a file instead of an environment variable?